web/styles/settings.css

```diff
@@ -396,8 +396,12 @@ td .button {
     }
 }
 
-.user_role {
-    min-width: 95px;
+.user_id {
+    min-width: 60px !important;
```

Comment:
We generally avoid using `!important`. Would be better to instead use a more specific selector.

Reply:
I have replaced it with a new ID. I'm not certain if this approach is correct or not. Please let me know if you have a better approach for this. Another approach that comes to mind is modifying the `app_components.css` file to include a check using `:not` to exclude applying the min-width to this specific column. However, I chose not to modify the `app_components.css` and instead added a new ID called `user_id_column`.